boards/silabs/radio_boards/xg23_rb4210a/doc/index.rst

```diff
@@ -72,24 +72,34 @@ Connections and IOs
 means Pin number 2 on PORTA, as used in the board's datasheets and manuals.
 
-+-------+-------------+-------------------------------------+
-| Name  | Function    | Usage                               |
-+=======+=============+=====================================+
-| PB2   | GPIO        | LED0                                |
-+-------+-------------+-------------------------------------+
-| PD3   | GPIO        | LED1                                |
-+-------+-------------+-------------------------------------+
-| PB1   | GPIO        | Push Button 0                       |
-+-------+-------------+-------------------------------------+
-| PB3   | GPIO        | Push Button 1                       |
-+-------+-------------+-------------------------------------+
-| PB0   | GPIO        | Board Controller Enable             |
-|       |             | VCOM_ENABLE                         |
-+-------+-------------+-------------------------------------+
-| PA8   | USART0_TX   | UART Console VCOM_TX US0_TX         |
-+-------+-------------+-------------------------------------+
-| PA9   | USART0_RX   | UART Console VCOM_RX US0_RX         |
-+-------+-------------+-------------------------------------+
++-------+-------------+---------------------------------------+
+| Name  | Function    | Usage                                 |
++=======+=============+=======================================+
+| PB2   | GPIO        | LED0                                  |
++-------+-------------+---------------------------------------+
+| PD3   | GPIO        | LED1                                  |
++-------+-------------+---------------------------------------+
+| PB1   | GPIO        | Push Button 0                         |
++-------+-------------+---------------------------------------+
+| PB3   | GPIO        | Push Button 1                         |
++-------+-------------+---------------------------------------+
+| PC3   | USART0_CLK  | Display SPI Clock (DISP_SCLK)         |
++-------+-------------+---------------------------------------+
+| PC1   | USART0_TX   | Display SPI Data (DISP_SI)            |
++-------+-------------+---------------------------------------+
+| PC8   | GPIO        | Display SPI Chip Select (DISP_SCS)    |
++-------+-------------+---------------------------------------+
+| PC6   | GPIO        | Display COM Inversion (DISP_EXTCOMIN) |
++-------+-------------+---------------------------------------+
+| PC9   | GPIO        | Display Enable (DISP_ENABLE)          |
++-------+-------------+---------------------------------------+
+| PB0   | GPIO        | Board Controller Enable               |
+|       |             | VCOM_ENABLE                           |
++-------+-------------+---------------------------------------+
+| PA8   | USART0_TX   | UART Console VCOM_TX US0_TX           |
++-------+-------------+---------------------------------------+
+| PA9   | USART0_RX   | UART Console VCOM_RX US0_RX           |
```

Comment:
In the DTS, `PA8` and `PA9` are declared for `EUSART0_TX` and `EUSART0_RX`.

Reply:
That's fixed in https://github.com/zephyrproject-rtos/zephyr/pull/83479. I assumed it would be merged first